app/src/main/java/de/westnordost/streetcomplete/data/quest/QuestController.kt

```diff
@@ -253,5 +253,7 @@ private val KEYS_THAT_SHOULD_NOT_BE_REMOVED_WHEN_SHOP_IS_REPLACED = listOf(
     // geometry
     "layer", "ele", "height", "area", "is_in",
     // notes and fixmes
-    "FIXME", "fixme", "note"
+    "FIXME", "fixme", "note",
+    // reference tags that are specific to the location and are unlikely to change when a business changes
+    "ref:bag", "ref:GB:uprn", "ref:linz:building_id", "ref:linz:topo50_id", "ref:ruian:.*", "ref:UrbIS"
```

Comment:
Ok. I trust you there, no idea what those tags are about.

Does the ruian tag really contain a `.*`? Looks like a regex

Reply:
I checked the ref page on the wiki to grab these :)